neonvm/runner/main.go

```diff
@@ -581,9 +584,22 @@ func main() {
 	// leading slash is important
 	cgroupPath := fmt.Sprintf("/%s-vm-runner", vmStatus.PodName)
 
-	if err := setCgroupLimit(qemuCPUs.use, cgroupPath); err != nil {
+	// To start up the VM as fast as possible, we let it run with a full CPU for the first few
+	// seconds (cpuThrottleDelay). Ideally we would get some kind of a signal from the VM when
+	// it's done all the endpoint-independent initialization, the 10 second delay is just a
+	// crude approximation of how long it will take. It doesn't need to be very accurate: if the
+	// delay is too long, we'll give away a few CPU seconds of extra computing power for free.
+	// If it's too short, the VM startup will just take a little longer.
+	log.Printf("initializing cgroup with full CPU")
+	if err := setCgroupLimit(1000, cgroupPath); err != nil {
```

Comment:
If the VM is going to have >1 CPU, we should let it have >1 (instead of limiting it at 1).

Alternatively: perhaps we just _don't_ set the cgroup limit for the first N seconds? If the slowdown happens to be in QEMU itself, then setting the cgroup limit to 1 CPU will still be slower than not having it.